packages/dhis2/src/Utils.js

```diff
@@ -53,8 +62,13 @@ export function ensureArray(data, key) {
   return Array.isArray(data) ? { [key]: data } : { [key]: [data] };
 }
 
-export function generateUrl(configuration, options, resourceType, path = null) {
-  let { hostUrl, apiVersion } = configuration;
+export function generateUrlPath(
```

Comment:
I think all this is _actually_ doing is maybe adding a version prefix to the path.

Like, it takes a path and just sticks a version number in front of it.

So maybe a better function name would be `prefixVersionToPath()` ?